v0.1/core/annotation.schema.json

```diff
@@ -15,49 +22,41 @@
       "description": "Metadata node identifier (id).",
       "pattern": "sands/v0.1/core/annotation/([A-Za-z0-9_-]+).json"
     },
-    "performedBy": {
-      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/CONTROLLED_TERM",
-      "description": "Responsible party that performed the annotation."
-    },
-    "anatomicalEntity": {
-      "$ref": "https://schema.hbp.eu/sands/core/anatomicalEntity.schema.json",
-      "description": "anatomicalEntity that is defined with annotation."
-    },
-    "criteriaType": {
-      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/CONTROLLED_TERM",
-      "description": "How was this annotation determined."
-    },
     "criteria": {
       "oneOf": [{
-        "$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v1.0/definitions.json/PROTOCOL"
+        "$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v3.0/definitions.json/PROTOCOL"
       }, {
-        "$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v1.0/definitions.json/PROTOCOL_EXECUTION"
+        "$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v3.0/definitions.json/PROTOCOL_EXECUTION"
       }],    
       "description": "criteria"
     },
-    "inspiredBy": {
-      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v3.0/definitions.json#definitions/FILE_INSTANCE",
-      "description": "inspiredBy"
+    "criteriaType": {
+      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/CONTROLLED_TERM",
+      "description": "How was this annotation determined."
     },
-    "annotationFile": {
-      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/ANNOTATION_FILE",
-      "description": "annotationFile"
+    "definingTerm": {
+      "$ref": "https://schema.hbp.eu/sands/core/anatomicalEntity.schema.json",
+      "description": "Anatomical entity that provides the term for the here specified annotation."
     },
-    "nativeCoordinateSpace": {
-      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/COORDINATE_SPACE",
-      "description": "The native coordinate space of this annotation."
+    "inspiredBy": {
+      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/IMAGE",
+      "description": "Link to (source) image(s) that inspired the definition of this annotation."
     },
```

Comment:
Issue #27 says count 0 - n for inspiredBy. Therefore, type needs to be array. In general, the types are missing for all properties. For objects it would be acceptable, but I think we agreed to be consisted and allows add all types, right?

Reply:
I'll change it to an array.

Adding the "object" type is indeed not needed for a valid schema. Not sure if we should do it. Nobody else is doing it, I think... Let use figure out best practice here later and then make it consistent in one PR for all schemas

Reply:
Okay. I just remembered sommething there, but I agree that it is pointless. Let do it like you suggested.